specs/waku/waku.md

```diff
@@ -222,9 +221,16 @@ The status message MUST contain an association list containing various options.
 
 This packet is used for sending the standard Waku envelopes.
 
-#### PoW Requirement
+#### Status Update
+
+The Status Update message is used to communicate an update of the settings of the node.
+The format is the same as the Status message, all fields are optional.
```

Comment:
Should we be precise about a case when the update is empty?

Reply:
Thats just a noop no? What other semantics could it reasonably have?

Reply:
It is noop. I asked a question wondering whether the spec should be explicit about this.

Reply:
Yes, and my follow up question is how you think it could be misinterpreted? We can be more explicit here, I just don't see what else it could mean. Are you worried it might be read as "reset"?

Reply:
I take it does not harm to be explicit about this, we can also say "an empty update MUST/SHOULD be ignored or considered a noop"

Reply:
sg